kernel/kernel-defaults/src/test/scala/io/delta/kernel/defaults/DeltaTableReadsSuite.scala

```diff
@@ -33,15 +33,18 @@ import io.delta.kernel.internal.util.InternalUtils.daysSinceEpoch
 import io.delta.kernel.types.{LongType, StructType}
 
 import org.apache.spark.sql.delta.{DeltaLog, DeltaOperations}
-import org.apache.spark.sql.delta.actions.{AddFile, Metadata}
-import org.apache.spark.sql.delta.implicits.stringEncoder
+import org.apache.spark.sql.delta.actions.AddFile
 
 import org.apache.hadoop.shaded.org.apache.commons.io.FileUtils
-import org.apache.spark.sql.functions.{col, current_timestamp, to_timestamp}
 import org.apache.spark.sql.functions.col
 import org.scalatest.funsuite.AnyFunSuite
 
-class DeltaTableReadsSuite extends AnyFunSuite with TestUtils {
+class LegacyDeltaTableReadsSuite extends AbstractDeltaTableReadsSuite with TestUtilsWithLegacyKernel
+
+// TODO: uncomment when new Kernel APIs supports ScanBuilder
```

Comment:
would it be a good happen to add a tracking issues for TODOs?  It would at least allow code search to make sure something isn't missed?

Reply:
I've been keeping track of some simportant followups and TODOs in this tracking issue: https://github.com/delta-io/delta/issues/4573

For this particular TODO, ScanBuilder API PR is already WIP -- https://github.com/delta-io/delta/pull/4663